Update README.md (#1)
halter73/tutorial: README.md

#### Expose the list of todo items

1. Above `await app.RunAsync();`, create a top-level method called `GetTodos` inside of the `Program.cs` file:
1. Above `await app.RunAsync();`, create a method called `GetTodos` inside of the `Program.cs` file:

    ```C#
    async Task<List<TodoItem>> GetTodos()
    {
        using var db = new TodoDbContext();

>         return await db.Todos.ToListAsync();

Reviewer: Did you mean to remove these spaces?

Author: Nope that was copy and paste fail

    await app.RunAsync();
    ```

    This method gets the list of todo items from the database and returns it. Returned values are written as JSON to the HTTP response.

> 1. Wire up `GetTodos` to the `api/todos` route by calling `MapGet`. This will happening before `await app.RunAsync();`:

Reviewer: ```suggestion
1. Wire up `GetTodos` to the `api/todos` route by calling `MapGet`. This should go before `await app.RunAsync();`:
```

    ```C#
    app.MapGet("/api/todos", (Func<Task<List<TodoItem>>>)GetTodos);
    app.MapPost("/api/todos", (Func<TodoItem, Task<StatusCodeResult>>)CreateTodo);

    await app.RunAsync();
    ```

> 1. Navigate to the `TodoReact` application which should be running on http://localhost:3000. Now, you will able to add new items. Refresh the TodoApi on http://localhost:5000 the page should show the stored todo items.

Reviewer: ```suggestion
1. Navigate to the `TodoReact` application which should be running on http://localhost:3000. Now, you will able to add new items. Behind the scenes when a new item is added, the `TodoReact` application makes a POST request to http://localhost:5000/api/todos which calls `CreateTodo` and stores the todo item in memory on the server. When the `TodoReact` application is refreshed, it makes a GET request to http://localhost:5000/api/todos calling `GetTodos` which should now return a non-empty JSON array containing the newly added items.
```
